index.html

```diff
@@ -44,7 +44,7 @@
   <body>
 
     <div id="root"></div>
-    <script src="index.js"></script>
+    <script src="index.tsx"></script>
```

Comment:
how does this work? I guess the typescript needs to be transpiled to javascript first (?)

Reply:
I also thought so, but it turned out that `parcel` automatically transpiles typescript without requiring any additional configuration, as shown [here](https://webcache.googleusercontent.com/search?q=cache:35EurI7sE-8J:https://parceljs.org/typeScript.html+&cd=11&hl=it&ct=clnk&gl=it).

Reply:
lovely! 👍